rollingpin/deploy.py

```diff
@@ -169,6 +177,10 @@ def signal_handler(sig, _):
                             except KeyError:
                                 raise ComponentNotBuiltError(component)
                             deploy_command.append(deploy_ref)
+
+                    # wait on ready command
+                    if not self.dangerously_fast:
```

Comment:
oh, i just realized, should this only be when we're doing a restart command? it's pretty meaningless otherwise right?